modules/report/js/commerce_pos_report.js

```diff
@@ -1,135 +0,0 @@
-(function ($) {
```

Comment:
I am not very sure where the old code could be used. I think we shouldn't keep the file unless its being referenced by any other code, I did a quick find in whole project and couldn't find this file being called anywhere